### DISCUSSION TITLE
MVN CLEAN INSTALL

### DISCUSSION THREAD

**Discussion by joseereyes** [sepinf-inc/IPED#2300] [Q&A]
Hi, I'm currently trying to start using this amazing software. I installed JDK 11 and everything required for building the app, but I got this error. Any  help with this?

![thumbnail_image](https://github.com/user-attachments/assets/5ca25218-8abf-47b0-b306-89c0176c188c)

@lfcnassif

**Comment by lfcnassif** [comment]
You also need to install JavaFX, it is stated on the project first page:
https://github.com/sepinf-inc/IPED?tab=readme-ov-file#building

We recommend to install Liberica Open JDK 11 Full package, standard package doesn't have JavaFx.
